src/modules/logicModule/LM_PC_RecurringPayments_v1.sol

```diff
@@ -297,6 +307,22 @@ contract LM_PC_RecurringPayments_v1 is
                     currentEpoch - currentPayment.lastTriggeredEpoch;
                 // If order hasnt been triggered this epoch
                 if (epochsNotTriggered > 0) {
+                    // assemble data array
+
+                    bytes32 flags;
+                    bytes32[] memory data;
+
+                    {
+                        bytes32[] memory paymentParameters = new bytes32[](3);
+                        paymentParameters[0] = bytes32(block.timestamp);
+                        paymentParameters[1] = bytes32(0);
```

Comment:
You dont need flag 2 here right?
the parameter is always zero so it should have standard values :thinking: